platforms/ios/lib/WysiwygComposer/Sources/HTMLParser/HTMLParserStyle.swift

```diff
@@ -51,18 +51,22 @@ public struct HTMLParserStyle {
 
     // MARK: - Internal
 
-    var codeBlockBackgroundStyle: BackgroundStyle {
-        BackgroundStyle(backgroundColor: codeBackgroundColor,
-                        borderColor: codeBorderColor,
-                        borderWidth: borderWidth,
-                        cornerRadius: cornerRadius)
+    var codeBlockStyle: BlockStyle {
+        BlockStyle(backgroundColor: codeBackgroundColor,
+                   borderColor: codeBorderColor,
+                   borderWidth: borderWidth,
+                   cornerRadius: cornerRadius,
+                   padding: 10.0,
+                   type: .background)
     }
 
-    var quoteBackgroundStyle: BackgroundStyle {
-        BackgroundStyle(backgroundColor: quoteBackgroundColor,
-                        borderColor: quoteBorderColor,
-                        borderWidth: borderWidth,
-                        cornerRadius: cornerRadius)
+    var quoteBlockStyle: BlockStyle {
+        BlockStyle(backgroundColor: quoteBackgroundColor,
+                   borderColor: quoteBorderColor,
+                   borderWidth: borderWidth,
+                   cornerRadius: cornerRadius,
```

Comment:
I guess both these should be 0.0 to draw properly

Reply:
Yeah, I will set them to 0.0 as default value, but maybe we could keep them to give more control to the user of the library on how to render them